test/testpy/__init__.py

```diff
@@ -71,13 +71,17 @@ def GetCommand(self):
       # inspector related tests). Also, if there is no ssl support the options
       # '--use-bundled-ca' and '--use-openssl-ca' will also cause a similar
       # failure so such tests are also skipped.
-      if ('--inspect' in flag[0] or \
-          '--use-bundled-ca' in flag[0] or \
-          '--use-openssl-ca' in flag[0]) and \
-          self.context.v8_enable_inspector == 0:
+      if (any(flag.startswith('--inspect') for flag in flags) and
+          not self.context.v8_enable_inspector):
+        print('Skipping as node was configured --without-inspector')
```

Comment:
Nit: `self.context.v8_enable_inspector` will be false also for `--without-ssl` and `--without-intl`
https://github.com/nodejs/node/blob/3212f77ac6d674a7d43c3303dcca22508c8be468/configure.py#L1526-L1530
So maybe chase the message to `'Skipping as node was compiler without inspector support.'`

Reply:
Ah yes, that makes sense. I'll update shortly. Thanks!